src/main/java/seedu/address/commons/core/Messages.java

```diff
@@ -10,5 +10,6 @@
     public static final String MESSAGE_INVALID_PERSON_DISPLAYED_INDEX = "The person index provided is invalid";
     public static final String MESSAGE_INVALID_EVENT_DISPLAYED_INDEX = "The event index provided is invalid";
     public static final String MESSAGE_PERSONS_LISTED_OVERVIEW = "%1$d persons listed!";
-
+    public static final String MESSAGE_INVALID_BLOCK_ROOM_LENGTH = "BlockRoom length must be 4!\nThe first character "
+            + "represents the block, the next 3 represents the room number";
```

Comment:
For the invalid message, maybe can abstract `"The first character represents the block, the next 3 {characters} represents the room number"` as `MESSAGE_INVALID_BLOCK_ROOM_FORMAT`. And display the error message when the format is wrong.

E.g. edit 1  br/3232
Error message: Block should be 1 of the following alphabets A, B, C, D
As mentioned in #211, it might be clearer to mention the correct format for `br` instead of the block.

Reply:
i cant parse the input if the length is correct. Updated my implementation to show constraints for both block and room